frontend/samples/rand.evy

```diff
@@ -0,0 +1,37 @@
+while true
+  print "╔════════════════════════╗"
+  print "║ Guess my number (1-10) ║"
+  print "╚════════════════════════╝"
+
+  n := 1 + (rand 10)
+  guess := readn
+
+  while guess != n
+    if guess > 10
+      print guess "Guess higher (1-10)."
```

Comment:
should this be "Guess lower" and the next "Guess higher"?

Reply:
:cookie:
thanks for the review!

Reply:
I fixed this up. 
Your comment re input focus is a really good one. also fixed.